lib/event/event-target.js

```diff
@@ -1,32 +1,98 @@
 "use strict";
 
+function flattenOptions(options) {
+    if (options !== Object(options)) {
+        return {
+            capture: !!options,
+            once: false,
+            passive: false
+        };
+    }
+    return {
+        capture: !!options.capture,
```

Comment:
Using `!!` for implicit type coercion has a few drawbacks

* it's easy for Future Contributor to make mistakes that do not look wrong at a glance (`!`, `!!`, `!!!`), which means it won't get caught be linters or reviewers
* it's difficult to understand for newcomers to JavaScript

Would you mind changing the `!!` to use the `Boolean()` function instead?

```js
  capture: Boolean(options.capture),
```